roles/foreman_repositories/tasks/main.yml

```diff
@@ -9,3 +9,9 @@
   tags:
     - packages
   when: ansible_os_family == 'RedHat'
+
+- name: 'Clean yum'
+  command: 'yum clean all'
```

Comment:
Can we use `yum clean expire-cache`? Perhaps there's even a native ansible way.

Reply:
I could not find a native Ansible way to do it.

Reply:
I can test expire-cache

Reply:
That would probably be adding `update_cache: true` here: https://github.com/theforeman/forklift/blob/871064fd382d3c8fa829f869e85089cdede17763/roles/foreman_repositories/tasks/redhat_release_repos.yml#L3-L5 but then the staging repos might need a similar treatment.